src/utils/showGameStatus.js

```diff
@@ -0,0 +1,23 @@
+import calculateWinner from './calculateWinner';
+
+const showGameStatus = (squares, xIsNext) => {
+    let winner = calculateWinner(squares);
+    let full = isNotEmpty(squares);
+
+    return winner ? `Winner: ${getCurrentPlayer(xIsNext)}` :
+           full ? `There's a DRAW` : `Next player: ${getNextPlayer(xIsNext)}`;
```

Comment:
Do not abuse the shorthand IF statements if you have to nest conditionals. Just use a common IF/ELSE statement in that case, or even better, write `guards` like in functional programming style:

```suggestion
    if (winner) return `Winner: ${getCurrentPlayer(xIsNext)}`
    if (!winner && full) return `There's a DRAW`
    
    return `Next player: ${getNextPlayer(xIsNext)}`
```